README.md

```diff
@@ -98,63 +108,56 @@ CSS
-or **[inversion-fixes.config](https://github.com/alexanderby/darkreader/blob/master/src/config/inversion-fixes.config) file**.
-- Click **edit** (sign-in to GitHub first).
-- **Insert your fix** there. Preserve **alphabetical order** by URL.
-- Provide a **short description** of what you have done.
-- Click **Propose file change**.
-- Review your changes. Click **Create pull request**.
-- The Travis CI will run tests reviewing your changes.
-- If you see a **red cross** click **Details** and see what is wrong and edit the existing Pull Request.
-- When you see a **green checkmark** then everything is fine.
-- Dark Reader developer will **review** and merge your changes making them available for all users.
+**Fixes for Filter and Filter+ modes**.
+- Filter and Filter+ work by inverting the whole web page and reverting necessary parts (images, videos, etc.), listed in the `INVERT` section.
+- If an inverted element contains images or other content that becomes wrongly displayed, `NO INVERT` rule can be used.
+- `REMOVE BG` removes the background image from an element and forces a black background.
 
-### Adding new features or fixing bugs
+### Add new features or fix bugs!
 
-If you would like to **add new feature** to Dark Reader or **fix a bug**, **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, wait for **approval**.
+If you would like to **add new feature** to Dark Reader or **fix a bug**, **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, and wait for **approval**.
 
-To build and debug the extension **install the [Node.js](https://nodejs.org/)** LTS.
-Install development dependencies by running `npm install` in the project root folder.
+To build and debug the extension **install the [Node.js](https://nodejs.org/)** LTS.  
+Install development dependencies by running `npm install` in the project root folder.  
 Then execute `npm run debug`.
 
 #### Chrome
-Open the `chrome://extensions` page.
-Disable the official Dark Reader version.
-Enable the **Developer mode**.
-Click **Load unpacked extension** button, navigate to the project's `debug/` folder.
+- Open the `chrome://extensions` page.
+- Disable the official Dark Reader version.
+- Enable the **Developer mode**.
+- Click **Load unpacked extension** button
+- Navigate to the project's `debug/` folder.
 
 #### Firefox
-Open `about:addons` page.
-Disable the official Dark Reader version.
-Open `about:debugging#addons` page.
-Click **Load Temporary Add-on** button, open `debug-firefox/manifest.json` file.
+- Open the `about:addons` page.
+- Disable the official Dark Reader version.
+- Open `about:debugging#addons` page.
+- Click **Load Temporary Add-on** button
+- Open the `debug-firefox/manifest.json` file.
+
+After making any code changes, the project will be automatically recompiled.  
```

Comment:
This is not more true sadly, so you can remove this.

Reply:
Sounds good. Just remove 138, or 135-138?

Reply:
Just Line 138

Reply:
Got it.  Will do.